libraries/core/lib/search/metaheuristics/SearchAlgorithm.ts

```diff
@@ -112,17 +111,17 @@ export abstract class SearchAlgorithm<T extends Encoding> {
 
     // Start initialization budget tracking
     budgetManager.initializationStarted();
-    getUserInterface().startProgressBar();
+    // getUserInterface().startProgressBar();
```

Comment:
We should remove these comments. We can connect the progress bar to the events

Reply:
Yes, already did that in #238. I wanted to keep the commented code in this PR to know where to put the events.